core/src/main/java/psiprobe/beans/ResourceResolverBean.java

```diff
@@ -308,10 +311,10 @@ private String getStringAttribute(MBeanServer server, ObjectName objectName,
    *
    * @return the global JNDI context
    */
-  protected javax.naming.Context getGlobalNamingContext() {
+  public static javax.naming.Context getGlobalNamingContext() {
 
     javax.naming.Context globalContext = null;
-    MBeanServer mbeanServer = getMBeanServer();
+    MBeanServer mbeanServer = ManagementFactory.getPlatformMBeanServer();
```

Comment:
@claudioweiler Smart...no reasont to call getter in same class to perform the same action.

Reply:
@hazendaz 
To use the getter it need to be changed to static too, has impact on interface and others implamentations.